pkgs/development/ocaml-modules/happy-eyeballs/lwt.nix

```diff
@@ -4,23 +4,29 @@
 , dns-client
 , logs
 , lwt
+, mtime
 }:
 
 buildDunePackage {
   pname = "happy-eyeballs-lwt";
+
   inherit (happy-eyeballs) src version;
 
-  buildInputs = [ cmdliner ];
+  minimalOCamlVersion = "4.08";
+
+  strictDeps = true;
 
-  propagatedBuildInputs = [
+  buildInputs = [
```

Comment:
![image](https://user-images.githubusercontent.com/5861043/191563564-765b6b2f-64f3-4319-90f1-0e0dc1c9c97d.png)

These should be at `propagatedBuildInputs`:
* dns-client
* happy-eyeballs
* logs
* lwt